### PR TITLE
Map tab

### PR DESCRIPTION
**Pull request by flacoman91**
adding a map tab and also adding a sweet tile map onto that.

### CODE REVIEW

File: src/MapPanel.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import DateIntervals from './DateIntervals';
+import Loading from './Dialogs/Loading'
+import { MemoryRouter } from 'react-router'
+import React from 'react'
+import TileChartMap from './TileChartMap';
+
+export class MapPanel extends React.Component {
+  render() {
+    return (
+      <MemoryRouter>
```
Comment (sephcoster) [inline src/MapPanel.jsx]: Using MemoryRouter here could prevent users from using their back / forward buttons and typically also does not update the application URL. Is that the intent for our the tab structure, or is this here mostly for testing / we plan to move this to a more conventional router at some point in the browser?

Comment (flacoman91) [reply src/MapPanel.jsx]: short answer, I don't know.  I used the original results panel as a baseline boilerplate and put put things in it to get it to work.   I thought that List, Trends and Map all have a similar structure and I did not know what Memory Router did

Comment (flacoman91) [reply src/MapPanel.jsx]: Ok, so it turns out MemoryRouter isn't needed here and was a big red herring.

The broken fwd/back functionality was introduced by new params I added in via the query reducer which caused the history to break.

File: src/reducers/query.jsx [hunk 2]
```diff
@@ -481,6 +516,12 @@ export function stateToQS( state ) {
     }
   } )
 
+  // exclude certain params from the URL since it's a calculated value
+  // coming from db and cant be restored
+  excludeParams.forEach( o => {
+    delete state[o]
+  } )
+
```
Comment (JeffreyMFarley) [inline src/reducers/query.jsx]: 👍

**Pull request merged**